### PR TITLE
CUSTOM-149: Fix liveness probe if custom replset name for 1.12.0

### PR DESCRIPTION
**Pull request by inelpandzic**
[CUSTOM-149](https://jira.percona.com/browse/CUSTOM-149) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

**CHANGE DESCRIPTION**
---
**Problem:**
*Short explanation of the problem.*

**Cause:**
*Short explanation of the root cause of the issue if applicable.*

**Solution:**
*Short explanation of the solution we are providing with this PR.*

**CHECKLIST**
---
**Jira**
- [ ] Is the Jira ticket created and referenced properly?
- [ ] Does the Jira ticket have the proper statuses for documentation (`Needs Doc`) and QA (`Needs QA`)?
- [ ] Does the Jira ticket link to the proper milestone (Fix Version field)?

**Tests**
- [ ] Is an E2E test/test case added for the new feature/change?
- [ ] Are unit tests added where appropriate?
- [ ] Are OpenShift compare files changed for E2E tests (`compare/*-oc.yml`)?

**Config/Logging/Testability**
- [ ] Are all needed new/changed options added to default YAML files?
- [ ] Are the manifests (crd/bundle) regenerated if needed?
- [ ] Did we add proper logging messages for operator actions?
- [ ] Did we ensure compatibility with the previous version or cluster upgrade process?
- [ ] Does the change support oldest and newest supported MongoDB version?
- [ ] Does the change support oldest and newest supported Kubernetes version?

### CODE REVIEW

File: e2e-tests/custom-replset-name/conf/some-name.yml [hunk 1]
```diff
@@ -0,0 +1,233 @@
+apiVersion: psmdb.percona.com/v1
+kind: PerconaServerMongoDB
+metadata:
+  name: some-name
+spec:
+  crVersion: 1.11.0
+  allowUnsafeConfigurations: true
+  backup:
+    enabled: true
+    image: percona/percona-server-mongodb-operator:1.11.0-backup
```
Comment (hors) [inline e2e-tests/custom-replset-name/conf/some-name.yml:10]:
```suggestion
    image: percona/percona-backup-mongodb:1.7.0
```

**Review by hors — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| cross-site-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/cross-site-sharded.log) |
| custom-replset-name | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/custom-replset-name.log) |
| storage | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/storage.log) |
| init-deploy | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/init-deploy.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/upgrade-consistency.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/one-pod.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/self-healing.log) |
| self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/self-healing-chaos.log) |
| demand-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/demand-backup.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/limits.log) |
| monitoring-2-0 | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/monitoring-2-0.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/scaling.log) |
| security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/security-context.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/operator-self-healing.log) |
| operator-self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/operator-self-healing-chaos.log) |
| rs-shard-migration | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/rs-shard-migration.log) |
| scheduled-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/scheduled-backup.log) |
| arbiter | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/arbiter.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/service-per-pod.log) |
| liveness | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/liveness.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/smart-update.log) |
| demand-backup-sharded | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/demand-backup-sharded.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/version-service.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/users.log) |
| data-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/data-sharded.log) |
| non-voting | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/non-voting.log) |
| upgrade | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/upgrade.log) |
| demand-backup-eks-credentials | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/demand-backup-eks-credentials.log) |
| upgrade-sharded | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-7-g2d6f212e/upgrade-sharded.log) |
| We run 29 out of 30|

commit: https://github.com/percona/percona-server-mongodb-operator/pull/1256/commits/2d6f212e1086c3a31bdae6efb97e5ae3fd2bdcdd
image: `perconalab/percona-server-mongodb-operator:PR-1256-v1.12.0-7-g2d6f212e`

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| storage | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/storage.log) |
| custom-replset-name | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/custom-replset-name.log) |
| cross-site-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/cross-site-sharded.log) |
| init-deploy | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/init-deploy.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/upgrade-consistency.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/one-pod.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/self-healing.log) |
| self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/self-healing-chaos.log) |
| demand-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/demand-backup.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/limits.log) |
| monitoring-2-0 | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/monitoring-2-0.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/scaling.log) |
| security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/security-context.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/operator-self-healing.log) |
| operator-self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/operator-self-healing-chaos.log) |
| rs-shard-migration | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/rs-shard-migration.log) |
| scheduled-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/scheduled-backup.log) |
| arbiter | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/arbiter.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/service-per-pod.log) |
| liveness | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/liveness.log) |
| demand-backup-sharded | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/demand-backup-sharded.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/smart-update.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/version-service.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/users.log) |
| data-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/data-sharded.log) |
| non-voting | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/non-voting.log) |
| demand-backup-eks-credentials | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/demand-backup-eks-credentials.log) |
| upgrade | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/upgrade.log) |
| upgrade-sharded | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-8-ga48d12cb/upgrade-sharded.log) |
| We run 29 out of 30|

commit: https://github.com/percona/percona-server-mongodb-operator/pull/1256/commits/a48d12cb289899877e8b2aa6f703a7fc3bc1c795
image: `perconalab/percona-server-mongodb-operator:PR-1256-v1.12.0-8-ga48d12cb`

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| cross-site-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/cross-site-sharded.log) |
| storage | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/storage.log) |
| init-deploy | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/init-deploy.log) |
| custom-replset-name | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/custom-replset-name.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/upgrade-consistency.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/one-pod.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/self-healing.log) |
| self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/self-healing-chaos.log) |
| demand-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/demand-backup.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/limits.log) |
| monitoring-2-0 | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/monitoring-2-0.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/scaling.log) |
| security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/security-context.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/operator-self-healing.log) |
| operator-self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/operator-self-healing-chaos.log) |
| rs-shard-migration | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/rs-shard-migration.log) |
| scheduled-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/scheduled-backup.log) |
| arbiter | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/arbiter.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/service-per-pod.log) |
| liveness | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/liveness.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/smart-update.log) |
| demand-backup-sharded | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/demand-backup-sharded.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/version-service.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/users.log) |
| data-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/data-sharded.log) |
| upgrade | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/upgrade.log) |
| non-voting | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/non-voting.log) |
| demand-backup-eks-credentials | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/demand-backup-eks-credentials.log) |
| upgrade-sharded | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-11-gc76ef02e/upgrade-sharded.log) |
| We run 29 out of 30|

commit: https://github.com/percona/percona-server-mongodb-operator/pull/1256/commits/c76ef02e5a4731b393daf94b6f0780214b841bb0
image: `perconalab/percona-server-mongodb-operator:PR-1256-v1.12.0-11-gc76ef02e`

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| storage | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/storage.log) |
| custom-replset-name | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/custom-replset-name.log) |
| init-deploy | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/init-deploy.log) |
| cross-site-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/cross-site-sharded.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/upgrade-consistency.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/self-healing.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/one-pod.log) |
| self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/self-healing-chaos.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/demand-backup.log) |
| monitoring-2-0 | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/monitoring-2-0.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/operator-self-healing.log) |
| operator-self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/operator-self-healing-chaos.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/scheduled-backup.log) |
| arbiter | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/arbiter.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/service-per-pod.log) |
| demand-backup-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/demand-backup-sharded.log) |
| liveness | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/liveness.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/smart-update.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/limits.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/scaling.log) |
| upgrade | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/upgrade.log) |
| security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/security-context.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/version-service.log) |
| rs-shard-migration | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/rs-shard-migration.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/users.log) |
| data-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/data-sharded.log) |
| non-voting | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/non-voting.log) |
| demand-backup-eks-credentials | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/demand-backup-eks-credentials.log) |
| upgrade-sharded | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-13-gdf7b2aec/upgrade-sharded.log) |
| We run 29 out of 30|

commit: https://github.com/percona/percona-server-mongodb-operator/pull/1256/commits/df7b2aec4895b68664b8987b8cd97c0fad4a07ad
image: `perconalab/percona-server-mongodb-operator:PR-1256-v1.12.0-13-gdf7b2aec`

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| storage | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/storage.log) |
| custom-replset-name | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/custom-replset-name.log) |
| cross-site-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/cross-site-sharded.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/init-deploy.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/upgrade-consistency.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/self-healing.log) |
| self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/self-healing-chaos.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/limits.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/demand-backup.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/scaling.log) |
| security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/security-context.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/operator-self-healing.log) |
| operator-self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/operator-self-healing-chaos.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/scheduled-backup.log) |
| rs-shard-migration | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/rs-shard-migration.log) |
| demand-backup-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/demand-backup-sharded.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/one-pod.log) |
| monitoring-2-0 | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/monitoring-2-0.log) |
| upgrade | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/upgrade.log) |
| upgrade-sharded | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/upgrade-sharded.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/pitr.log) |
| arbiter | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/arbiter.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/service-per-pod.log) |
| pitr-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/pitr-sharded.log) |
| liveness | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/liveness.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/smart-update.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/version-service.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/users.log) |
| data-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/data-sharded.log) |
| non-voting | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/non-voting.log) |
| demand-backup-eks-credentials | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1256/v1.12.0-14-gfdc9db0a/demand-backup-eks-credentials.log) |
| We run 31 out of 30|

commit: https://github.com/percona/percona-server-mongodb-operator/pull/1256/commits/fdc9db0af86240e9165f39eae8260128951afbea
image: `perconalab/percona-server-mongodb-operator:PR-1256-v1.12.0-14-gfdc9db0a`

**Review by hors — APPROVED** [review]

**Pull request merged**